test/source/tests/browser-unit-tests/unit-ContactStore.js

```diff
@@ -118,26 +116,38 @@ BROWSER_UNIT_TEST_NAME(`ContactStore.search ignores top-level domains and common
   return 'pass';
 })();
 
-BROWSER_UNIT_TEST_NAME(`ContactStore.search allows searching for name parts that look like top-level domains and common domains`);
+BROWSER_UNIT_TEST_NAME(`ContactStore.search allows searching for name parts that look like top-level domains and common domains`); // prettier-ignore
```

Comment:
Is this because of backticks?
I suggest to add support for ordinary single quotes instead.
We can replace backtick with single quotes

Reply:
We should probably ignore whole test/source/tests/browser-unit-tests folder. It's not really javascript nor typescript.

Reply:
Could be length (149 chars). Do you think we should just ignore the length here, if it eslint works ok for the rest?

Reply:
I added `prettier-ignore` here as it was automatically wrapping text, so test case name was on the next line and it broke test file parsing at https://github.com/FlowCrypt/flowcrypt-browser/blob/feature/issue-4526-eslint/test/source/tests/unit-browser.ts#L68

Reply:
> We should probably ignore whole test/source/tests/browser-unit-tests folder. It's not really javascript nor typescript.

Yes, I disabled `eslint` in this folder, text wrapping was done by `prettier`

Reply:
> I added `prettier-ignore` here as it was automatically wrapping text, so test case name was on the next line and it broke test file parsing at https://github.com/FlowCrypt/flowcrypt-browser/blob/feature/issue-4526-eslint/test/source/tests/unit-browser.ts#L68

If we allow length of 190 (as it was before), there would be much less changes which is good as it'd be easier to `git blame` etc.

Reply:
190 seems to be too much, as it makes some expressions quite difficult to read.

I think setting 120 as max line length will lead to better readability and encourage us to split complex expressions into simpler ones. For sure, there are some cases when `prettier` makes strange decisions about code wrap, but we can fix them manually or just disable `prettier` for some lines to produce code without any formatting.

There are too many format changes in this PR because I ran `prettier` across all files to make code look consistent, but all future code will be automatically formatted for any developer before commit.

By the way, it's not really 'max-length' rule, but our desired line length, as `prettier` documentation says:

> For readability we recommend against using more than 80 characters:
> 
> In code styleguides, maximum line length rules are often set to 100 or 120. However, when humans write code, they don’t strive to reach the maximum number of columns on every line. Developers often use whitespace to break up long lines for readability. In practice, the average line length often ends up well below the maximum.
> 
> Prettier’s printWidth option does not work the same way. It is not the hard upper allowed line length limit. It is a way to say to Prettier roughly how long you’d like lines to be. Prettier will make both shorter and longer lines, but generally strive to meet the specified printWidth.
> 
> Remember, computers are dumb. You need to explicitly tell them what to do, while humans can make their own (implicit) judgements, for example on when to break a line.
> 
> In other words, don’t try to use printWidth as if it was ESLint’s [max-len](https://eslint.org/docs/rules/max-len) – they’re not the same. max-len just says what the maximum allowed line length is, but not what the generally preferred length is – which is what printWidth specifies.

@rrrooommmaaa are you okay with trying to use `120` as our preferred line length?

Reply:
I suspect before this refactor, there were still many lines far beyond 190. The limit used to be 240 on this codebase when it only had one contributor, because that contributor had screen more wide than tall :-)

I suppose a print-width of 160 would be a reasonable compromise. A lot of developers chose one or two-word method/variable names, but for sake of clarity (thus security), some of our methods and variables are quite wordy, to make it _very_ clear what is being done and how. An 80 limit would be unusable on our codebase. I agree with Roman that 120 is still quite small target for this particular codebase.

If this is an automated process reformatting it, I propose to try 160. Then we'll evaluate it. If Roma finds it still too dense, we can try 150, or 140 as an alternative.

Reply:
160 is okay for me, I pushed commit with increased `printWidth` value